### DISCUSSION TITLE
What Payment System Could I Install In A Joomshopping?

### DISCUSSION THREAD

**Discussion by Pinovski** [joomla/joomla-cms#34911] [Q&A] [answered]
What Payment System Could I Install In A Joomshopping?

I am working on a store whose client uses JoomShopping, he needs a payment system. I have been suggested to install https://www.sequra.es/en this system, however, it seems to me that it is not compatible. Any idea or alternative to install a payment system in JoomShopping?

Regards

**Comment by Fedik** [comment, marked as answer]
Please contact JoomShopping developers.
